db/migrations/20230307102539_create_user_table.sql

```diff
@@ -0,0 +1,17 @@
+-- migrate:up
+CREATE TABLE users (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  password VARCHAR(200) NOT NULL,
+  name VARCHAR(50) NOT NULL,
+  email VARCHAR(200) NOT NULL,
+  phone_number VARCHAR(200) NOT NULL,
+  address VARCHAR(3000) NOT NULL,
+  birth VARCHAR(200) NOT NULL,
+  gender VARCHAR(50) NOT NULL,
+  point DECIMAL NOT NULL,
```

Comment:
decimal의 경우 자릿수를 지정해 주어여 합니다. decimal에 사용법을 한 번 더 확인해 주시면 좋을 것 같습니다.
```suggestion
  point DECIMAL(12,2) NOT NULL,
```